src/test/java/seedu/address/logic/commands/UnlabelCommandTest.java

```diff
@@ -0,0 +1,126 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_LABEL;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.HashSet;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.label.Label;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.TagName;
+import seedu.address.testutil.TagBuilder;
+
+public class UnlabelCommandTest {
+    private ModelStubWithTag modelStub;
+
+    @BeforeEach
+    public void setUp() {
+        Tag newTag = new TagBuilder().build();
+        modelStub = new ModelStubWithTag(newTag);
+    }
+
+    @Test
+    public void equals() {
+        TagName oldTagName = new TagName("oldTag");
+        TagName newTagName = new TagName("newTag");
+
+        HashSet<Label> emptyLabels = new HashSet<>();
+        HashSet<Label> labels = new HashSet<>();
+        labels.add(new Label("testLabel"));
+
+        UnlabelCommand firstUnlabelCommand = new UnlabelCommand(newTagName, emptyLabels);
+        UnlabelCommand secondUnlabelCommand = new UnlabelCommand(oldTagName, labels);
```

Comment:
Add another UnlabelCommand with same label but different TagName for 100% coverage.

Test for this case:
`this.tagName.equals(((UnlabelCommand) other).tagName))`

Reply:
Okay. I have added the new test.